src/components/BirdAlertList.tsx

```diff
@@ -110,7 +121,7 @@ export default function BirdAlertList() {
                                         <CommandEmpty>No statuses found.</CommandEmpty>
                                         <CommandGroup>
                                             {allStatuses.map((framework) => {
-                                                if (framework === "Delivered") {
+                                                if (framework === "Delivered" || framework === "Released On Site" || framework === "Incomplete") {
```

Comment:
can you rename `framework` to `status`? I think that was some bad copypasta on my part.

Reply:
I had considered removing them from the allStatuses list, but I wasn't sure which way was "best practice" because If I remove them the reason it still wont show the bird alerts is I think because it technically gets a type error, since the status is not in allStatuses. Not sure if its really an issue though to remove them